README.adoc

```diff
@@ -104,7 +104,7 @@ $ oc new-project MY_PROJECT_NAME
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
-$ oc import-image fis-java-openshift:2.0 --from=registry.access.redhat.com/jboss-fuse-6/fis-java-openshift:2.0 --confirm
+$ oc import-image fuse-java-openshift:2.0 --from=registry.access.redhat.com/jboss-fuse-7/fuse-java-openshift:2.0 --confirm
```

Comment:
jboss-fuse-7 repository doesn't exist, it was renamed to fuse7.
Also tag 2.0 does't exist

Reply:
@luigidemasi Thanks for noticing this. I'll use this information in general fix of thses issues (https://issues.redhat.com/browse/ENTESB-13020)